pkg/plugins/runtime/gateway/testdata/http/26-gateway-route.yaml

```diff
@@ -49,7 +51,7 @@ Clusters:
         envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
           '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
           commonHttpProtocolOptions:
-            idleTimeout: 3600s
```

Comment:
It's because ExternalService should be routed though Egress but there is no egress. BEcause of this meshContext has no protocol for this cluster and that's why we can see timeout change

Reply:
Which part of this PR exactly causes this?

Reply:
Introduction of new MeshTimeout policies. Because there is a policy but has not matching protocol we are not defining a timeout. https://github.com/kumahq/kuma/blob/master/pkg/plugins/policies/meshtimeout/plugin/xds/configurer.go#L130 maybe we need a fallback for other protocols